src/Modules/ModParser.lua

```diff
@@ -1739,6 +1740,28 @@ local specialModList = {
 		flag("ChaosCanIgnite", { type = "Condition", var = "IgnitingConflux" }),
 	},
 	["gain chilling, shocking and igniting conflux for %d seconds"] = { },
+	["you have igniting, chilling and shocking conflux while affected by glorious madness"] = {
+		flag("PhysicalCanChill", { type = "Condition", var = "ChillingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("LightningCanChill", { type = "Condition", var = "ChillingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("FireCanChill", { type = "Condition", var = "ChillingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("ChaosCanChill", { type = "Condition", var = "ChillingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		mod("EnemyIgniteChance", "BASE", 100, { type = "Condition", var = "IgnitingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("PhysicalCanIgnite", { type = "Condition", var = "IgnitingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("LightningCanIgnite", { type = "Condition", var = "IgnitingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("ColdCanIgnite", { type = "Condition", var = "IgnitingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("ChaosCanIgnite", { type = "Condition", var = "IgnitingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		mod("EnemyShockChance", "BASE", 100, { type = "Condition", var = "ShockingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("PhysicalCanShock", { type = "Condition", var = "ShockingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("ColdCanShock", { type = "Condition", var = "ShockingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("FireCanShock", { type = "Condition", var = "ShockingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		flag("ChaosCanShock", { type = "Condition", var = "ShockingConflux" }, { type = "Condition", var = "AffectedByGloriousMadness" }),
+	},
+	["immune to elemental ailments while affected by glorious madness"] = {
+		mod("AvoidChill", "BASE", 100, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		mod("AvoidFreeze", "BASE", 100, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		mod("AvoidIgnite", "BASE", 100, { type = "Condition", var = "AffectedByGloriousMadness" }),
+		mod("AvoidShock", "BASE", 100, { type = "Condition", var = "AffectedByGloriousMadness" }),
```

Comment:
Need to also add lines for `AvoidScorch`, `AvoidBrittle` and `AvoidSap`

Reply:
Good catch. I just copied these from other "immune to elemental ailments" sections. Could probably do a pass on those overall.

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/d600d809c6ef3b5df0e604c2626d088135fa9867/src/Modules/ModParser.lua#L1619-L1624

https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/d600d809c6ef3b5df0e604c2626d088135fa9867/src/Modules/ModParser.lua#L1817-L1822

Reply:
Yeah realised we don't handle avoidance for the alt ailments so can exclude it from this PR too. Would probs be good to do a pass in a future PR and add it to all the relevant mods